test/model/test_version_history.py

```diff
@@ -1,16 +1,16 @@
+from datetime import datetime as dt
+
 import pytest
-from mock import patch, call, MagicMock, PropertyMock, Mock
 from dateutil import parser
 from dateutil.tz import tzutc
-from datetime import datetime as dt
+from mock import MagicMock, Mock, PropertyMock, call, patch
 
-from dafni_cli.model import model, version_history
-from test.fixtures.model_fixtures import (
-    get_models_list_fixture,
-    get_single_model_fixture,
-    get_model_metadata_fixture,
-)
 from dafni_cli.consts import TAB_SPACE
+from dafni_cli.model import model, version_history
+
+from test.fixtures.model_fixtures import (get_model_metadata_fixture,
+                                          get_models_list_fixture,
+                                          get_single_model_fixture)
```

Comment:
This would change with black formatting:
```suggestion
from test.fixtures.model_fixtures import (
    get_model_metadata_fixture,
    get_models_list_fixture,
    get_single_model_fixture,
)
```